Mathlib/Analysis/Matrix.lean

```diff
@@ -148,22 +148,22 @@ instance [StarAddMonoid α] [NormedStarGroup α] : NormedStarGroup (Matrix m m 
   ⟨norm_conjTranspose⟩
 
 @[simp]
-theorem nnnorm_col (v : m → α) : ‖col v‖₊ = ‖v‖₊ := by
+theorem nnnorm_col {ι : Type*} [Unique ι] (v : m → α) : ‖col ι v‖₊ = ‖v‖₊ := by
```

Comment:
Could you please use `variable {ι : Type*} [Unique ι]` to avoid the repetition?

Reply:
done!